packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -790,8 +791,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private latestSummaryGCVersion: GCVersion;
     // This is the source of truth for whether GC is enabled or not.
     private readonly shouldRunGC: boolean;
+    // This is the runtime version when doc is first created
+    private readonly createDocRuntimeVersion: string;
+    // This is the time stamp of when doc is first created
+    private readonly createDocTimeStamp: number;
+    // This is the counter of summary
```

Comment:
This comment does not give any extra information than the variable name. This is the counter of the last summary that happened in this client or something like that would be more informative. Also, the variable name could be similar - `lastSummaryCount`